packages/api-client/src/client.ts

```diff
@@ -0,0 +1,97 @@
+import type { ErrorResponse } from '@translation/api-types';
+import Languages from './languages';
+
+export { ErrorResponse };
+export * from './languages';
+
+export interface ApiClientOptions {
+  baseUrl: string;
+}
+
+export interface ApiClientRequestOptions {
+  method: string;
+  path: string;
+  query?: Record<string, string>;
+  body?: unknown;
+}
+
+export type ApiClientGetOptions = Pick<
+  ApiClientRequestOptions,
+  'path' | 'query'
+>;
+export type ApiClientPatchOptions = Pick<
+  ApiClientRequestOptions,
+  'path' | 'query' | 'body'
+>;
+export type ApiClientPostOptions = Pick<
+  ApiClientRequestOptions,
+  'path' | 'query' | 'body'
+>;
+
+export class ApiClientError extends Error {
```

Comment:
Then this error will make it easy to capture errors in frontend code, and the error itself will be typed with an error code and any future fields on errors